typeface-sourcecodepro-materialdesign

```diff
@@ -19,3 +19,4 @@
 #define typeface_bar_glyph_battery_unknown 
 #define typeface_bar_glyph_time 
 #define typeface_bar_glyph_workspace 
+#define typeface_bar_glyph_thermometer 
```

Comment:
Here it is: 

(FYI I used `gucharmap` to view all characters of a given font and copied from there.): 

![image](https://user-images.githubusercontent.com/49683/70407381-f9197380-19f8-11ea-811c-af3e009613c9.png)

Reply:
Ahh that's a great tool. Thanks for sharing it. I swapped the glyph :)